website/docs/cli/state/index.html.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: "docs"
+page_title: "Manipulating State - Terraform CLI"
+---
+
+# Manipulating Terraform State
+
+Terraform relies on [state data](/docs/state/index.html) to remember which
+real-world object corresponds to which managed resource in the configuration;
+this allows it to modify an existing object when changes are made to the
+corresponding resource declaration. For infrastructure managed by multiple
+people, the state data must be made available to all collaborators using a
+[backend](/docs/configuration/backend.html) that stores state remotely.
+
+Under normal conditions, Terraform updates state automatically during plans and
+applies. However, it's sometimes necessary to make deliberate adjustments to
+Terraform's state data, usually to compensate for changes to the configuration
+or the real managed infrastructure.
+
+When such situations arise, you can use one of Terraform CLI's commands for
+inspecting and modifying state data.
+
+Most of these commands are designed to help with a specific type of situation.
+
+~> **Important:** Modifying state data outside a normal plan or apply is
+inherently dangerous, because it can cause Terraform to lose track of managed
+resources. Depending on your provisioning practices and infrastructure usage,
+losing track of resources might cost you a lot of money, annoy all of your
+colleagues, or even compromise the security of your operations. Make sure to
+keep backups of your state data when modifying state out-of-band.
+
+## Checking Things Out
+
+- Use [the `terraform state list` command](/docs/commands/state/list.html) and
+  [the `terraform state show` command](/docs/commands/state/show.html) to
+  inspect state data before using any of the other state commands to do anything
+  risky. (Or just use them when you're curious about something.)
+- Use [the `terraform refresh` command](/docs/commands/refresh.html) to update
+  state data to match the real-world condition of the managed resources. This is
+  done automatically during plans and applies, but not during any of the direct
+  state operations listed below, so you might want to refresh before reading or
+  modifying state if it's been a few hours since the last plan or apply.
+
+## Importing Pre-existing Resources
+
+The `terraform import` command can associate unmanaged resources with a resource
+address in a configuration, bringing them under Terraform's control.
+
+Importing is documented in its own section of the Terraform CLI docs. See
+[Importing Infrastructure](/docs/import/index.html) for details.
+
+## Forcing Re-creation of Resources (Tainting)
```

Comment:
This and the following sections feel like they have a _lot_ of info in them for a section overview. Can we be more brief and move some of this to the command pages (or remove it if it's already there?)

Reply:
So, the problem I'm trying to solve here is this: understanding the state commands enough to even _know which command to investigate further_ is honestly kind of hard. Reading back through this, maybe I did go a bit overboard :P but I do think out-of-band state manip is an obscure enough topic that the overview _does_ need to be relatively meaty. 

Something that might help would be sorting the state commands into subsections in the nav itself, so the categories are more apparent right off the bat. That might help reinforce this info enough that we could shrink it down some.

Reply:
I think that would help. There's some info here about how the commands work and I think we can probably cover most of them by saying what they do, with a sentence or two about why you would use them for the more obscure ones? 

If you want I can do a real edit of this page; just let me know if you want me to after you try the organize-y thing.